.travis.yml

```diff
@@ -0,0 +1,88 @@
+# Use trusty for better performance (and avoiding mysql/postgres/solr gone issues on precise and container infra)
+dist: trusty
+sudo: required
+
+language: php
+
+services:
+  - mysql
+
+# Mysql isn't installed on trusty (only client is), so we need to specifically install it
+addons:
+  apt:
+    packages:
+    - mysql-server-5.6
+    - mysql-client-core-5.6
+    - mysql-client-5.6
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+env:
+  global:
+    # For functional and acceptance tests
+    - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
+    # In order to specify extra flags like ignoring platform reqs, using only stable packages and so on.
+    - COMPOSER_FLAGS=""
+    - EZPLATFORM_REPO="https://github.com/ezsystems/ezplatform.git"
+
+matrix:
+  fast_finish: true
+  include:
+# 7.1
+    - php: 7.1
+      env: TEST_CONFIG="phpunit.xml"
+    - php: 7.1
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+# 7.2
+    - php: 7.2
+      env: TEST_CONFIG="phpunit.xml"
+    - php: 7.2
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+# 7.3
+    - php: 7.3
+      env: TEST_CONFIG="phpunit.xml"
+    - php: 7.3
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+    - name: "Code Style Check"
```

Comment:
Please set explicitly php 7.3 for this job for performance. Job itself shows something that looks like warning
> This job ran using the default PHP version, which was updated to 7.2 on March 12th, 2019

so I'd say it's not best practice to have implicit version.